app/views/shared/_newsletter.html.erb

```diff
@@ -0,0 +1,20 @@
+<section>
+  <div class="container-fluid">
+    <div class="jumbotron" align="middle">
+      <h2>Join our newsletter</h2>
+      <div class="lead">
+        <p>Be the first one to know about our next events</p>
+      </div>
+      <form>
+        <input type="text" id="input_first_name" class="form-control" placeholder="First Name" required="" autofocus="">
+        <input type="text" id="input_first_name" class="form-control" placeholder="Last Name" required="" autofocus="">
+        <input type="email" id="input_email" class="form-control" placeholder="Email address" required="" autofocus="">
+        <div>
+          <label>
+            <input type="button" value="Subscribe">
+          </label>
```

Comment:
I'm not sure what the purpose of the div>label>input elements is here. As the rest of our site is looking v v bootstrappy, could we steal [their example of a button too](http://getbootstrap.com/css/#forms)?